python/cucim/src/cucim/skimage/morphology/tests/test_skeletonize.py

```diff
@@ -65,3 +65,77 @@ def test_compare_skimage(self, invert):
         result = thin(cp.asarray(h))
         expected = thin_cpu(h)
         assert_array_equal(result, expected)
+
+
+class TestMedialAxis():
+    def test_00_00_zeros(self):
+        '''Test skeletonize on an array of all zeros'''
```

Comment:
nit: just wondering if there is a reason to use `'''` instead of `"""` in this file.

Reply:
Yeah we should probably move to `"""`. This is what [`black`]( https://github.com/psf/black ) does

Reply:
Happy to change this to """ and we can probably apply black at some point, but I didn't do it so far for two reasons
1.) It messes with readability of various small test array definitions unless `# fmt: off`, etc. are placed around them
2.) easier to compare diffs with upstream scikit-image without it.